patch/PAK1_UTF8/15627_0000.txt

```diff
@@ -6,10 +6,10 @@ get_chat_time=><x17><x15><xC5><x24><x02><x89><x80>
 
 [ENDBLOCK]
 <Woodrow>, Master <Alba> and <Chelsea>...
-They were all so nice. 
+Were both such nice people. 
 [ENDBLOCK]
 Consider yourself lucky this time. Not everyone
-you meet is as good of a person. 
+you meet is as magnanimous as they are. 
```

Comment:
"Magnanimous" doesn't fit here, given the context. Would change to "Not everyone
you meet is kind-hearted/well-intentioned."